app/services/attendance_csv_importer.rb

```diff
@@ -40,6 +42,8 @@ def process_row(row)
     @row = row
     @child = child
 
+    return unless (@start_date..@end_date).cover?(@row['check_in'].in_time_zone(@child.timezone).at_beginning_of_day)
```

Comment:
Date range condition applied here in order to handle the timezone. Can be applied earlier in the process row block, but the child wouldn't necessarily exist and then we wouldn't be able to find the appropriate timezone which could be relevant.

Reply:
This is definitely the right place to do this!